recipe/meta.yaml

```diff
@@ -218,10 +217,10 @@ outputs:
         - scipy
 
     test:
-      imports:
-        - ray.rllib
       commands:
-        - rllib --help
+        - python -m pip install gym     # [not win]
+        - python -c "import ray.rllib"  # [not win]
+        - rllib --help                  # [not win]
```

Comment:
this basically means `ray-rllib` _package_ in conda-forge is broken... why is using `gym` from conda-forge not enough for it?..

Reply:
The conda builds of gym are stuck on gym 0.21.0 since versions beyond that depend on pygame and ale-py, which are not yet packaged for conda-forge. We could pin `gym <=21.0`, but then that would preclude #75, since there is no gym for aarch64 and ppc64le. Between `pip install` ing gym and having a aarch64 build, I would think it would be preferable to have an aarch64 build for cloud platforms.

Reply:
In rethinking this, it seems better to leave the broken gym as it was for the 2.0.0 release. I think in the long-term the dependency will move to gymnasium, and perhaps it will be easier to get that properly built for conda forge.

I reverted the change from a conda dependency to a pip dependency.